### DISCUSSION TITLE
Ability to inspect converted messages in studio

### DISCUSSION THREAD

**Discussion by MattOslin** [foxglove/community#687] [Q&A]
It would be very handy to inspect converted messages in studio, similar to the raw messages panel. This would allow users to see that their conversions make sense, or even use converters to translate complicated message formats into easy to read formats to look at in studio.

**Comment by foxymiles** [comment]
Slack discussion here:
https://foxglove.slack.com/archives/C028UEY858S/p1681344513140339

Specifically this pertains to message converters: https://foxglove.dev/docs/studio/extensions/extension-context#registermessageconverter

**Reply by jtbandes** [reply]
Internal tracking issue: https://linear.app/foxglove/issue/FG-1039
